src/components/TrashLookup.jsx

```diff
@@ -17,16 +17,14 @@ const TrashLookUp = props => {
   const [selectedAddress, setSelectedAddress] = useState("");
   const [isAutoTextHidden, setIsAutoTextHidden] = useState(false);
 
-  const addressData = () => {
-    let searchQuery = _.trim(selectedAddress);
-    let filtered = PostData;
-
-    if (searchQuery.length > 0) {
-      filtered = PostData.filter(
-        m => m.address1.toLowerCase().indexOf(searchQuery.toLowerCase()) > -1
-      );
-    }
-    return { data: filtered };
+  const getAddress = (query = "") => {
```

Comment:
This should really go in a service, since this will change to an actual ajax request in the future.